config/environments/uat.rb

```diff
@@ -111,6 +111,9 @@
   # Feature toggle for subscriptions in MMT
   config.subscriptions_enabled = true
 
+  # Feature toggle for UMM-T
```

Comment:
I think we should talk with PO to determine when we want to flip this.  I'd like to get a ticket number and put it in a comment here (not necessarily in this ticket, but an action for a future PR)